disco_aws_automation/disco_iam.py

```diff
@@ -382,9 +382,13 @@ def _get_trust_relationship_json(self, policy):
             json_data = json.load(tr_file)
             return json.dumps(json_data, indent=4)  # indentation is important
 
-    def _create_role_name(self, role_prefix, policy, naked_roles):
+    def _create_role_name(self, role_prefix, policy, naked_roles=None):
```

Comment:
I added the code in the method itself to derive the naked_roles from the config.